lib/collections/users/collection.js

```diff
@@ -1,34 +1,73 @@
 Meteor.users.helpers({
-  fullName() {
-    return `${this.profile.firstName} ${this.profile.lastName}`;
-  },
-
-  email() {
-    return this.emails[0].address || this.profile.publicEmail;
-  },
-
-  profilePicture() {
-    var image = Images.findOne({ '_id': this.profile.image });
-
-    if (image)
-      return image;
-    else {
-      var defaultPictureIndex = UserUtils.getDefaultPictureIndex(this._id);
-      return { url: `/images/default_${defaultPictureIndex}.png` };
-    }
+    fullName() {
+        return `${this.profile.firstName} ${this.profile.lastName}`;
+    },
+
+    email() {
+        return this.emails[0].address || this.profile.publicEmail;
+    },
+
+    profilePicture() {
+        var image = Images.findOne({
+            '_id': this.profile.image
+        });
+
+        if (image)
+            return image;
+        else {
+            var defaultPictureIndex = UserUtils.getDefaultPictureIndex(this._id);
+            return {
+                url: `/images/default_${defaultPictureIndex}.png`
+            };
+        }
 
-  },
+    },
+    allAnswersCount(userId) {
+        var answersCount = 0;
+        var bestAnswersCount = 0;
+        var questions = Questions.find({}).fetch();
+        questions.forEach(function(question) {
+            var answers = question.answers
+            for (var i = 0; i < answers.length; i++) {
+                var answer = Answers.find({
+                    _id: answers[i]
+                }).fetch();
+                if (answer[0].ownerId == userId) {
```

Comment:
I here mean 0 because the `.fetch()` function returns an array of documents and since it can return only one document I use 0 directly

makes sense?